roles/openshift_logging_curator/files/actions.yaml

```diff
@@ -0,0 +1,37 @@
+# ---
+# Remember, leave a key empty if there is no value.  None will be a string,
+# not a Python "NoneType"
+#
+# Also remember that all examples have 'disable_action' set to True.  If you
+# want to use this action as a template, be sure to set this to False after
+# copying it.
+# actions:
+#   1:
+#     action: delete_indices
```

Comment:
This is based on @portante suggestions?  we only support 'delete' now but could support 'close' or other in future?

Reply:
elastic introduced support for config files https://www.elastic.co/guide/en/elasticsearch/client/curator/current/actionfile.html